### PR TITLE
Feature/backup gitea mongo mlflow

### PR DESCRIPTION
**Pull request by aacecandev**
This PR is linked to this issue https://github.com/konstellation-io/kdl-server/issues/652

### CODE REVIEW

File: backup/README.md [hunk 1]
```diff
@@ -1,46 +1,120 @@
...
-* Stop Gitea stateFulSet
-  
-`kubectl -n kdl delete statefulset gitea`
+Projects, users and roles are being stored in a Mongo pod. The bindings are stored in the `kdl` db, under the `projects` and `users` collections. Although it would be enough to have data backed up, we also dumped the `admin` database in order to achieve consistency with all the references that the admin user has in each project because it's unique `ObjectId`.
 
-* Deploy a pod from where perform the restore
+## Backup Gitea DB
 
-`kubectl -n kdl apply -f restore-pod.yaml`
+Science Toolkit Team were doing this backup using a `CronJob` and they were using a **manual process to restore it** using an ad-hoc pod and `psql` commands. The Kubernetes manifests can be found in `helm/templates/gitea/restore-pod.yml`.
```
Comment (blopezpi) [inline backup/README.md:76]: Science Toolkit? Is this Konstellation, no?

File: backup/README.md [hunk 2]
```diff
@@ -1,46 +1,120 @@
...
-* Download the required backup from S3 Bucket
-  
-`aws s3 cp s3://$BUCKET_NAME/backup_gitea_<date>.tar.gz .`
+### Backup MLFlow
 
-* Unzip backup
+The relevant files to be backed up are kept in:
 
-`tar zxvf backup_gitea_<date>.tar.gz`
+- MLFlow DB
```
Comment (blopezpi) [inline backup/README.md:95]: This one is available only in our environment.

File: helm/kdl-server/crds/project-operator-crd.yaml [hunk 3]
```diff
@@ -130,7 +130,7 @@ spec:
                 properties:
                   enabled:
                     description: Boolean to specify if TLS is enabled
-                    type: string
+                    type: boolean
```
Comment (blopezpi) [inline helm/kdl-server/crds/project-operator-crd.yaml:133]: And this one?

**Review by blopezpi — APPROVED** [review]
LGTM

**Review by alpiquero — APPROVED** [review]

**Pull request merged**